public class FindFacorial{
  public static void main(String[] args){
    try{
      Calc_Factorial.findFact(5);
    }
    catch(Exception e){
      System.out.println("Exception is :"+e);
    }
  }
}
